lib/omniauth/openid_connect/util.rb

```diff
@@ -0,0 +1,45 @@
+# coding:utf-8
+# frozen_string_literal: true
+
+module OmniAuth
+  module OpenIDConnect
```

Comment:
Why do you define all these methods directly under the `OmniAuth::OpenIDConnect` scope? Shouldn't it be something like `OmniAuth::OpenIDConnect::Utils` module?

Reply:
Moved `OmniAuth::OpenIDConnect::Util`